File: Service_eHospital_DongNai_A/Implement/DM_BenhVienTuyenDuoiService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class DM_BenhVienTuyenDuoiService : BaseService<DM_BenhVienTuyenDuoi, IDM_BenhVienTuyenDuoiRepository>
    , IDM_BenhVienTuyenDuoiService
    {
    private readonly IDM_BenhVienTuyenDuoiRepository _DM_BenhVienTuyenDuoiRepository;
    public DM_BenhVienTuyenDuoiService(IDM_BenhVienTuyenDuoiRepository DM_BenhVienTuyenDuoiRepository) : base(DM_BenhVienTuyenDuoiRepository)
    {
    _DM_BenhVienTuyenDuoiRepository = DM_BenhVienTuyenDuoiRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/DM_BenhVienTuyenDuoi.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DM_BenhVienTuyenDuoi } from './DM_BenhVienTuyenDuoi.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DM_BenhVienTuyenDuoiService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BenhVien_Id', 'MaBenhVien', 'TenBenhVien', 'TenBenhVien_En', 'TenBenhVien_Ru', 'DiaChi', 'Hang_Id', 'Loai_Id', 'Tuyen_Id', 'CapQuanLy_Id', 'TinhThanhPho_Id', 'TamNgung', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id', 'SoDienThoai', 'Email'];

    List: DM_BenhVienTuyenDuoi[] | undefined;
    ListFilter: DM_BenhVienTuyenDuoi[] | undefined;
    FormData!: DM_BenhVienTuyenDuoi;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DM_BenhVienTuyenDuoi";
    }
}

